overlord/overlord.go

```diff
@@ -383,6 +384,11 @@ func (o *Overlord) CommandManager() *cmdstate.CommandManager {
 	return o.cmdMgr
 }
 
+// ConfigManager returns the manager responsible for doing configuration
```

Comment:
missing the period,  seems a couple other here also slipped through without :/

Reply:
Thanks, I fixed this (and the ones above) now as well.